beetmoverscript/templates/maven_geckoview.yml

```diff
@@ -0,0 +1,62 @@
+---
+metadata:
+    name: "Maven repository"
+    description: "Maps artifacts to spec'd maven location"
+    owner: "release@mozilla.com"
+
+s3_bucket_path: maven2/org/mozilla/{{ artifact_id }}/{{ version }}/  # Maven groupId is org.mozilla
+
+mapping:
+{% for locale in ['en-US'] %}
+  "{{ locale }}":   # Locale is not needed for geckoview, it's used by move_beets, though
+{% for product in ['geckoview'] %}
```

Comment:
Tiny nit: can we add a secondary tab here to indent properly the manifest so that's easier to see it's a nested `for`?

Reply:
Sure. Done in https://github.com/mozilla-releng/beetmoverscript/pull/163/commits/bb88611b5a78aecfcb45f7ea4c897946bd485718